app/routes/customer_routes.py

```diff
@@ -0,0 +1,74 @@
+from app import db
+from flask import Blueprint, jsonify, request
+from app.models.customer import Customer
+from app.models.rental import Rental
+from app.models.video import Video
+from app.models.helper import invalid_cust_data, invalid_customer
+
+customers_bp = Blueprint("customers", __name__, url_prefix="/customers")
+
+@customers_bp.route("", methods=["POST"])
```

Comment:
👍 

Nice job splitting your endpoints into separate functions. And having the additional helper functions for validating ids especially useful. Consider extending the helpers that validate existing ids to actually retrieve the record as well.